console/src/main/scala/io/shiftleft/console/cpgcreation/CpgGeneratorFactory.scala

```diff
@@ -19,7 +19,7 @@ class CpgGeneratorFactory(config: ConsoleConfig) {
     guessLanguage(inputPath)
       .flatMap { l =>
         report("Using generator for language: " + l)
-        cpgGeneratorForLanguage(l, config.frontend, config.install.rootPath.path)
+        cpgGeneratorForLanguage(l, config.frontend, config.install.rootPath.path, additionalArgs = Nil)
```

Comment:
it's already `additionalArgs: List[String] = List()` ?

Reply:
Thanks for the comments! The `cpgGeneratorForLanguage` method [does not give a default value](https://github.com/ShiftLeftSecurity/codepropertygraph/blob/master/console/src/main/scala/io/shiftleft/console/cpgcreation/package.scala#L16) for `additionalArgs`, hence the explicit values for these.